fixes deprecation warnings for 0.4

rened opened the pull request:
tests pass locally for 0.3 and 0.4 (tested on osx)

simonster commented:
Do you have real tests that you could add there? ATM, the only thing in `test/DICOM_test.jl` is a `using` statement. Even if not, I have no qualms with merging this, as the changes seem pretty innocuous and this is totally broken on 0.4 otherwise.

simonster commented on REQUIRE:
```diff
@@ -0,0 +1 @@
+Compat
```
Should also require `julia 0.3`

rened replied:
Good point, added!

rened commented:
Added a test (didn't see that there are none...). This started to unearth a few needed updates for 0.4. I fixed what got triggered by the test image, but I assume there are other fixes needed...

rened commented:
Added a `.travis.yml`, could you please enable it in the travis settings?

ihnorton commented:
@rened, sorry for missing this. Travis is now enabled, and commits rebased/squashed in 346e20027c867936027264dc6a9f7cf7af30fc9a with `--author=...` as you. Thanks!

(I removed Compat support, but could put it back if you need it...)

ihnorton commented:
If it would make sense to move this repo somewhere else, let me know and I would be happy to do so. A colleague and I need this and will be working on it, but the repo certainly doesn't have to be under my account.

rened commented:
@ihnorton no problem at all! ;-) I only need to read dicoms from time to time, and sometimes need to revert to using dcm4che anyway, as some of the dicoms can't be loaded. I tried to look into this but didn't get very far. I am totally fine with keeping the package here!
